qtrio/_tests/test_core.py

```diff
@@ -73,6 +73,32 @@ def test():
     result.assert_outcomes(passed=1)
 
 
+def test_reenter_event_writes_to_stderr_for_exception(capsys, testdir):
+    test_file = r"""
+    import pytest
```

Comment:
Is `import pytest` needed here?

Reply:
Nope, forgot to remove it when refactoring this file-in-a-string.